lib/graphql/schema_comparator/changes.rb

```diff
@@ -414,11 +434,31 @@ def initialize(old_schema, new_schema)
         end
 
         def message
-          "Schema mutation root has changed from `#{old_schema.mutation}` to `#{new_schema.mutation}`"
+          "Schema mutation root has changed from `#{old_schema.mutation.graphql_name}` to `#{new_schema.mutation&.graphql_name}`"
```

Comment:
Same question about the safe operator usage here

Reply:
So here, I think the null safe operator is necessary—this basically covers the case where someone removes a mutation root altogether (instead of changing/renaming it). I just verified that this was also a possible breakage on the schema query/subscription type changed classes, so I added it in there as well. I figured, it would be better to just make these changes null-safe, rather than add a set of `SchemaQueryTypeRemoved`/`SchemaMutationTypeRemoved`/`SchemaSubscriptionTypeRemoved` changes as well.

Let me know if that makes sense to you as well—thank you for taking a look!

Reply:
Another alternative is unifying these into "root operation" changes:

* `RootOperationTypeRemoved`
* `RootOperationTypeAdded`
* `RootOperationTypeChanged`

Messaging could be "Schema root operation type `mutation` changed from {} to {}`".

What do you think?

Reply:
Yeah that makes a lot of sense. Just put those changes in—thanks for the review/suggestion!